libs/dh/metering-point/feature-search/src/routes.ts

```diff
@@ -60,6 +60,13 @@ export const dhMeteringPointRoutes: Routes = [
                 (m) => m.DhMeteringPointMasterDataComponent
               ),
           },
+          {
+            path: getPath<MeteringPointSubPaths>('messages'),
+            loadComponent: () =>
+              import('@energinet-datahub/dh/metering-point/feature-overview').then(
+                (m) => m.DhMeteringPointMessagesComponent
+              ),
+          },
```

Comment:
question: should we put it behind a feature flag or it's ready for prime time :)

Reply:
Good question, I checked up on it and it is good to go (since it is currently only available to admins) 🙂